gatsby/content/projects/bridges/matrix-bifrost.mdx

```diff
@@ -3,18 +3,29 @@ layout: projectimage
 title: matrix-bifrost
 categories: 
  - bridge
-description: bridging with a variety of backends including libpurple and xmpp.js
+description: bridging with xmpp.js
 author: Half-Shot
 maturity: Beta
 language: TypeScript
 license: Apache-2.0
 repo: https://github.com/matrix-org/matrix-bifrost
 room: "#bifrost:half-shot.uk"
 featured: true
-bridges: libpurple
-thumbnail: /images/libpurple.png
+bridges: XMPP
+thumbnail: /images/xmpp.png
 ---
 
-General purpose puppeting bridges using libpurple and other backends.
+&#10003; Plain Text  
+&#10003; Formatted Messages  
+&#10003; Media/files  
+&#10003; Redactions/Deletions  
+&#10003; Replies  
+&#10003; Group/1:1 Chats  
+&#10003; transmit roomnick to XMPP (only aria-net.org)  
+&#10007; Calls
+
+You can self host [matrix-bifrost](https://github.com/matrix-org/matrix-bifrost) in order to bridge your rooms
```

Comment:
I don't think this is a great description of what the bridge does, and I don't think we should include off-matrix.org information about where you can get the bridge.

That isn't to say we can't have hosting information on matrix.org, but I don't want to conflate software projects with hosted instances.

Reply:
> I don't think this is a great description of what the bridge does

The old one was better? :thinking: 
> I don't think we should include off-matrix.org information about where you can get the bridge.

You mean like this one which i used as template?
https://matrix.org/bridges/#discord

> That isn't to say we can't have hosting information on matrix.org, but I don't want to conflate software projects with hosted instances.
okay then should the aria-net.org bridge should be listed separately because it's advanced more than the matrix.org bridge and doesn't use libpurple anymore?